buildSrc/build.gradle.kts

```diff
@@ -98,17 +98,30 @@ gradlePlugin {
     }
 }
 
```

Comment:
Why these changes enable unconditionally the test? The old code left it to the caller to enable it or not, and now the `-PenablePluginTests` wouldn't work, right?

Reply:
If you look at line 117-120, it still retains the old functionality of only activating if the property is set. These new tasks I'm registering do *not* get called automatically when tests are ran. They have to be explicitly called. They exist so you can just call the task itself if you wanna run plugin tests, versus calling test with the `enablePluginTests` parameter. Although both methods work totally fine still.

It's worth noting that `enablePluginTest` is basically the same thing as `enableTests` (this is true of the current impl and the previous impl). We just disable all buildSrc tests unless the param is specified. Here, I offer an alternative task you can call instead- which allows your to use gradle tasks windows and such instead of needing to find the param for enabling tests.

Reply:
Got it, thanks for the clarification!